scripts/set-rn-version.js

```diff
@@ -38,7 +38,12 @@ let argv = yargs
 
 const buildType = argv.buildType;
 const version = argv.toVersion;
-validateBuildType(buildType);
+
+try {
+  validateBuildType(buildType);
+} catch (e) {
+  throw e;
+}
```

Comment:
Effectively, @cortinico noticed that this should be the default behavior... @kelset do you think we can remove this try-catch?

Reply:
during my testing, adding this seemed to have an effect so that exec() would catch the error correctly. I can remove but honestly, what's the problem with having it?